### PR TITLE
Update INSTALL.md, ENABLE_DATA set to ON by default

### PR DESCRIPTION
**Pull request by eyvallah**
Some changes the macosx build instruction and add ENABLE_DATA to supported parameters.

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -274,23 +274,17 @@ Install Homebrew
 Install required packages:
 
 ```
-brew install cmake cspice fmt gettext libpng lua qt5 jpeg eigen
+brew install cmake glew fmt gettext libpng lua qt5 jpeg eigen
```
Comment (375gnu) [inline INSTALL.md:277]: why did you remove cspice?

Comment (eyvallah) [reply INSTALL.md:277]: i think it is optional, i'll list it in optional later

Comment (eyvallah) [reply INSTALL.md:277]: done
